nemo/collections/nlp/data/dialogue/dataset/dialogue_s2s_generation_dataset.py

```diff
@@ -42,21 +42,22 @@ def __init__(self, dataset_split: str, dialogues_processor: object, tokenizer, c
         if self.cfg.debug_mode:
             self.features = self.features[:16]
 
-    def format_actions(self, actions):
+    @staticmethod
+    def format_actions(prompt_template, actions):
         actions_str = []
         for action in actions:
             act = action['act'].lower()
             slot = action['slot']
             value = action['values'][0] if action['values'] else ''
 
-            if self.cfg.prompt_template == 'values':
+            if prompt_template == 'values':
```

Comment:
same comment - please make sure this if eventually ends with "else"

Reply:
sure!